Makefile

```diff
@@ -7,6 +7,7 @@ SERVICES = users things http coap lora influxdb-writer influxdb-reader mongodb-w
 	bootstrap opcua authn twins mqtt provision
 DOCKERS = $(addprefix docker_,$(SERVICES))
 DOCKERS_DEV = $(addprefix docker_dev_,$(SERVICES))
+DOCKER_IMAGE_NAME_PREFIX ?= ""
```

Comment:
I would use `mainflux` as prefix here

Reply:
I've thought about it but I'm confused about what to do with `/`: 
should be `/` part of `DOCKER_IMAGE_NAME_PREFIX` or not.

Reply:
I prefer to keep it outside but let's see what others think about

Reply:
@manuio I think `mainflux` should be default, and then `<docker_registry_url>/mainflux` can be used for other cases.

I preferred previous implementation, looks cleaner and more configurable.

Reply:
@drasko I'm no sure to understand. I proposed `DOCKER_IMAGE_NAME_PREFIX ?= "mainflux"`
and `$(DOCKER_IMAGE_NAME_PREFIX)/$(svc)` to simplify an eventual renaming and because it's hard-coded for many commands.

Reply:
Ah, cool - then we are on the same page.

Reply:
Good, I made `MF_DOCKER_IMAGE_NAME_PREFIX ?= mainflux/`

Reply:
We proposed `mainflux`, not `mainflux/` ;)

Reply:
ok, updated:

MF_DOCKER_IMAGE_NAME_PREFIX ?= mainflux